### PR TITLE
Add "Deleting a response" article

### LINKED ISSUE #295: Add article about trashing a response

### PR DESCRIPTION
**Pull request by kariannemah**
Closes #295.

### CODE REVIEW

File: contents/articles/screendoor/responses/deleting_responses.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Deleting a response
+order: 9
+app_pages:
+  - projectadmin-responses-index
+---
+
+If you need to remove duplicate or erroneous responses, you can send them to the trash. At any time, you can choose to restore these responses. Or you can  delete them permanently, removing from your project all traces of their data.
```
Comment (ajb) [inline contents/articles/screendoor/responses/deleting_responses.md]: Could there be other reasons they want to remove them? Do we want to make the assumption that it's one of these two reasons?

File: contents/articles/screendoor/responses/deleting_responses.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Moving a response to the trash
+
+From your project’s Responses page, use the checkboxes on the left side of the table to select the responses you want to send to the trash. Press the More button above the table, and select Delete from the dropdown.
+
+![Sending responses from the response table to the trash.](../images/delete_1.png)
+
+You can also send a response to the trash while you are reading it. In the lower right-hand side of the page, click "Delete response."
```
Comment (ajb) [inline contents/articles/screendoor/responses/deleting_responses.md]: s/while you are reading it/from its detailed view

Or do we call it "the response page" anywhere?

File: contents/articles/screendoor/responses/deleting_responses.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+![Sending an individual response to the trash.](../images/delete_2.png)
+
+To view responses in the trash, visit your project’s Responses page and click Trash in the left sidebar.
+
+![Viewing responses in the trash.](../images/delete_3.png)
+
+### Restoring a response from the trash
+
+First, visit your project’s Responses page and click Trash in the left sidebar. Then, you can use the checkboxes on the left side of the table to select the responses you would like to restore. Press the Restore button above the table. These responses will now appear in your table of responses.
```
Comment (ajb) [inline contents/articles/screendoor/responses/deleting_responses.md]: Last sentence is weird. How about "now appear in your _main_ table"?

File: contents/articles/screendoor/responses/deleting_responses.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+You can also permanently delete a response while you are reading it. In the lower right-hand side of the page, click "Delete forever."
+
+![Permanently deleting an individual response.](../images/delete_7.png)
+
+---
+
+## F.A.Q.
+
+### How do I block a specific person from submitting to my form?
```
Comment (ajb) [inline contents/articles/screendoor/responses/deleting_responses.md]: :+1:

File: contents/articles/screendoor/responses/deleting_responses.md (hunk 1 above)
Comment (ajb) [inline contents/articles/screendoor/responses/deleting_responses.md]: An idea: could we focus this introduction more on the _benefits_ of the trash page, as well as a brief high-level concept to help users understand how it works? Something like:

> When deleting response inside of your Screendoor project, they will go to the trash before being permanently destroyed. This allows you to remove duplicate or erroneous responses without fear of losing data if you make a mistake.

**Comment by ajb** [comment]
Looks solid. Comments inline

**Comment by kariannemah** [comment]
Thanks! Made the edits you suggested.

**Comment by ajb** [comment]
Great!

**Pull request merged**